ruby/game/game.rb

```diff
@@ -0,0 +1,234 @@
+# ----- PSEUDOCODE
+# Input word
+  # Loop until guess_count > word.length + 3
+    # Display word as spaces + guessed char (guessed_word)
+    # Input guess (char)
+    # Compare guess to word
+    # Check if char already guessed
+        # If so, prompt again
+    # Increment guess_count by 1
+    # Loop through array of word char's
+      # If guess = char in word, modify guessed_word at index
+        # Print correct guess
+      # Else
+        # Print incorrect guess
+    # Break if no '_' left in guessed_word array
+      # Declare 
+# Print either win or lose
+  # Based on won_game boolean
+
+class Game
+  attr_accessor :word, :guess_count, :guess, :guessed_word, :guessed_letters, :is_over, :won_game, :already_guessed
+  
+  def initialize 
+    @word = []
+    @guess = ""
+    @guessed_letters = []
+    @already_guessed = true
+    @guessed_word = []
+    @guess_count = 0
+    @is_over = false
+    @won_game = true
+  end
+
+  def input_word(word)
+    @word = word.split('')
```

Comment:
I feel that word should be passed in on initialize and be kept as a string.  You can always just split it in local class methods when you need too.